runners/katacoda/templates/createProject.md

```diff
@@ -0,0 +1,16 @@
+<%= text; %>
+
+## Setting up your java project
+
+<%= cdCommand; %>
+
+Navigate to the 'workspaces/main/' folder in your devonfw installation directory.
+`cd workspaces/main/`{{execute}}
+
+Now you can use devon to setup a <%= language; %>  project for you by executing the following command.
```

Comment:
either write "use the devon command" or say "use devonfw to setup" as devonfw is a trademark. We need to get the usage clear in our docs

Reply:
Should I also use the cd Template to change to workspace/main or should I keep it that way?

Reply:
yes would be better as maybe you want to create the project at a completely different folder